deposit_management/data/ir_model_fields.xml

```diff
@@ -86,4 +99,65 @@ self['x_deposit_categ_id'] = self.env['product.category'].search([('x_is_a_depos
         <field name="readonly" eval="True"/>
         <field name="store" eval="False"/>
     </record>
+    <record id="account_tax_group_is_deposit" model="ir.model.fields">
+        <field name="ttype">boolean</field>
+        <field name="field_description">Is Deposit</field>
+        <field name="model_id" ref="account.model_account_tax_group"/>
+        <field name="name">x_is_deposit</field>
+    </record>
+    <record id="account_tax_deposit_field" model="ir.model.fields">
+        <field name="name">x_is_deposit</field>
+        <field name="field_description">Is Deposit</field>
+        <field name="model_id" ref="account.model_account_tax"/>
+        <field name="related">tax_group_id.x_is_deposit</field>
+        <field name="ttype">boolean</field>
+        <field name="readonly" eval="True"/>
+    </record>
+    <record id="sale_order_line_total_deposits_field" model="ir.model.fields">
+        <field name="name">x_total_deposits</field>
+        <field name="field_description">Deposits</field>
+        <field name="ttype">monetary</field>
+        <field name="model_id" ref="sale.model_sale_order_line"/>
+        <field name="depends">product_uom_qty, tax_ids</field>
+        <field name="compute"><![CDATA[for record in self:
+        deposit = sum(tax.amount for tax in record.tax_ids if tax.x_is_deposit)
+        record['x_total_deposits'] = deposit * record.product_uom_qty]]></field>
```

Comment:
I think, depends should be product_uom_qty, tax_ids.amount and tax_ids.x_is_deposit.